mlir/test/Pass/scf2cf-print-liveness-crash.mlir

```diff
@@ -0,0 +1,19 @@
+// RUN: mlir-opt %s -pass-pipeline="builtin.module(func.func(convert-scf-to-cf), func.func(test-print-liveness))"
```

Comment:
Nit : Can we make filename more generic? We could add other tests other than crash
`scf2cf-print-liveness-crash.mlir` -> `scf-to-cf-and-print-liveness.mlir`

Reply:
Sure, fixed